package.json

```diff
@@ -14,18 +14,15 @@
     "unseed": "yarn sequelize-cli db:seed:undo:all",
     "build": "babel src -d lib",
     "start": "yarn run build && cross-env NODE_ENV=production node lib/index.js",
-    "dev": "cross-env NODE_ENV=development nodemon --exec babel-node -r dotenv/config src/index.js",
+    "dev": "cross-env NODE_ENV=development nodemon --exec babel-node src/index.js",
     "lint": "eslint src/** test/** --fix",
     "prepare": "husky install",
-    "test": "yarn migrate --env test && cross-env NODE_ENV=test c8 --reporter=text --reporter=lcov ./node_modules/.bin/mocha --timeout 200000 --exit --require @babel/register"
-  },
-  "nyc": {
-    "include": "src",
-    "exclude": "node_modules"
+    "test": "yarn migrate --env test && cross-env NODE_ENV=test c8 mocha --timeout 100000 --exit --require @babel/register"
```

Comment:
why remove the test script from the scripts list?

Reply:
the script was not removed actually. It was edited to move coverage options along with the nyc property, from package.json to .nycrc.json (extends package.json).